### ISSUE TITLE
Integrate  JDToaster

### ISSUE THREAD

**Issue by jdisho** [jdisho/Papr#88]
Replace alerts with [JDToaster](https://github.com/jdisho/JDToaster).

**Comment by artyom2206** [comment]
Can I take it?

**Comment by jdisho** [comment]
Yes, you can, but let’s discuss it first in this issue @artyom2206.

**Comment by artyom2206** [comment]
So i installed it to Papr and implemented instead of each alertAction, but there is a problem with JDToaster on new IOS 13.2 version. When the toast shows from the bottom, then it disappears(check GIF). Toast duration also not works, I called it manually(IOS 12.4). Should I do pull request?

(https://user-images.githubusercontent.com/47772683/69388436-25c74e80-0cc9-11ea-935a-3855fe85161a.gif)

**Comment by jdisho** [comment]
This certainly does not look as I expected 🤔 
Would you like to investigate the issue in the JDToaster and create a PR?

**Comment by artyom2206** [comment]
Ok, I will look at it

**Comment by jdisho** [comment]
Adding a strong reference to the `UIWindow` will solve the issue. @artyom2206

**Comment by jdisho** [comment]
The issue is fixed! I will do a release soon.

**Comment by Dishant10** [comment]
Hey @jdisho 
Is this issue already fixed ? If not I'd like to contribute on that. Let me know at the earliest.

**Comment by jdisho** [comment]
I don't know if this is relevant. As you may see, I'm not maintaining this app anymore 😴
